scripts/export_comments.py

```diff
@@ -62,24 +62,31 @@ def create_comments_excel_file(survey_id, submissions):
 
     for submission in submissions:
         comment = submission.data['data'].get('146')
+
+        boxes_selected = ""
+        for key in ['146a', '146b', '146c', '146d', '146e', '146f', '146h']:
```

Comment:
146a will be the yes/no to significant changes. Since this will always yes in order for comments to be given in the first place we don't need to include here.

Reply:
It's possible that we may go beyond `h` for other surveys so worth having a think about an easy way to extend this. 
Possibly:
```
from string import ascii_lowercase

for key in ('146' + letter for letter in ascii_lowercase[1:]):
```

or to save the import
`for key in ('146' + letter for letter in 'bcdefghijklmnopqrstuvwxyz'):`

Reply:
Went for the acii_lowercase one, thought it looked tidier

Reply:
and removed the 146a too!